lib/edamam_api_wrapper.rb

```diff
@@ -0,0 +1,32 @@
+class EdamamApiWrapper
+  class RecipeError < StandardError ; end
+
+  APP_KEY = ENV["APP_KEY"]
+  APP_ID = ENV["APP_ID"]
+  BASE_URL = "https://api.edamam.com/search?"
+  URI_BASE = "http://www.edamam.com/ontologies/edamam.owl#recipe_"
+  FROM = 1
```

Comment:
`BASE_URL` and `URI_BASE` are very visually similar, and it's not immediately clear how they're different. Could you rename one or both to something more distinct?